homeassistant/components/simplisafe/__init__.py

```diff
@@ -257,6 +258,25 @@ def _async_get_system_for_service_call(
     raise ValueError(f"No system for device ID: {device_id}")
 
 
+@callback
+def _async_log_deprecated_service_call(
```

Comment:
Nit: This is not an async method.  Should it have the `_async` prefix?

Reply:
It's a fairly common pattern in HASS to prefix methods with async that are async-safe. Since that's adopted elsewhere in this integration, I kept things consistent.

Reply:
👍🏻